src/main/java/xyz/aadev/generitech/client/gui/machines/GuiPulverizer.java

```diff
@@ -89,10 +91,6 @@ public void drawBG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
             if (tileEntity.getPower() > 48000) powerLevel = 25;
             if (tileEntity.getPower() == 0) powerLevel = -1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00122)